src/code42cli/cmds/securitydata.py

```diff
@@ -362,84 +337,92 @@ def search(
     use_checkpoint,
     saved_search,
     or_query,
+    columns,
     include_all,
     **kwargs,
 ):
 
     """Search for file events."""
-    output_header = try_get_default_header(
-        include_all, _create_search_header_map(), format
-    )
-    formatter = FileEventsOutputFormatter(format, output_header)
-    cursor = _get_cursor(state, use_checkpoint)
+    if format == FileEventsOutputFormat.CEF and columns:
+        raise click.BadOptionUsage("--columns option can't be used with CEF format.")
```

Comment:
this method takes the args (option name, message) so we you want the message to print the console I think it should be like -> 
`raise click.BadOptionUsage(columns, "--columns option can't be used with CEF format.")`